### ISSUE TITLE
Divide destinations from nodes.

### ISSUE THREAD

**Issue by loopRW** [hulop/NavCog2#41]
Destinations as nodes are problematic when too many destinations are on a same logical edge. Should we start changing to POI based destinations?

**Comment by dsato80** [comment]
I agree with POI based destinations.
I'm worrying about backward compatibility with this function.
If we choose this we should not keep previous way, otherwise code will be confusing I guess

**Comment by loopRW** [comment]
I agree that we should not keep both. Are there any concerns for backwards compatibility? The current maps can be ported by placing a destination POI on what are currently destination nodes. That way we can upgrade all the environments we have.
